core/spdb/entity.go

```diff
@@ -111,6 +111,7 @@ type MigrateGVGUnitMeta struct {
 	DestSPID                 uint32
 	LastMigratedObjectID     uint64
 	MigrateStatus            int // scheduler assign unit status.
+	RetryTime                int //
```

Comment:
Modifying the db schema may cause incompatible upgrade issues. How do you plan to deal with this issue?

Reply:
For non-prod env, the automigration can be perform, for prod env, need DBA to run migration first.